### DISCUSSION TITLE
Build CBC with nauty

### DISCUSSION THREAD

**Discussion by verflieger** [coin-or/Cbc#645] [Q&A] [answered]
How can I build CBC with symmetry detection support via nauty? Can anyone please provide some details? If I build it via coinbrew, I get the message `configure: error: Cannot find symbol(s) nauty with NTY` during configure. I cannot find any details in the docs.

**Comment by svigerske** [comment, marked as answer]
On https://github.com/coin-or/Cbc/blob/stable/2.10/.github/workflows/linux-ci.yml, they install package `libnauty2-dev` and use `--with-nauty-incdir=/usr/include/x86_64-linux-gnu/nauty --with-nauty-lib="-L/usr/lib/ -lnauty"` for Ubuntu.

**Reply by verflieger** [reply]
Thank you, with this I can use orbital branching 👍
